promising/promising_function.py

```
@@ -386,10 +385,12 @@ async def protected_run(
             return await promise
         finally:
-            if await_children is RECURSIVELY:
-                await promise.await_children(recursively=True)
-            elif await_children:
-                await promise.await_children(recursively=False)
+            if await_children is WHOLE_SUBTREE:
+                await promise.await_children(whole_subtree=True)
+            elif await_children is True:
+                await promise.await_children(whole_subtree=False)
+            elif await_children is not False:
+                raise ValueError(f"Invalid await_children={await_children!r}; expected WHOLE_SUBTREE, True, or False")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)